### ISSUE TITLE
Green tick button doesn't activate through keyboard

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11257]
## Expected Behavior

Green tick button should activate through keyboard

## Actual Behavior

1. Navigate to `Home` on the left navigation. Make sure `Components` is selected on menu below the main heading.
2. Select `all` filter and in the table navigate to any application under name column with Runtime resources tab available (see screenshot below).
3. In the next page that opens up for selected application, please navigate to `Runtime resources` tab.
4. In the `resources table`, navigate to `green tick` button.
5. Verify whether green tick button is getting activated through keyboard or not.
6. Same thing happens on `all` filter under `Your company` text.

![image](https://user-images.githubusercontent.com/54085737/166472290-039f781e-1073-4c1c-bd50-772eaa8e56a7.png)

## Context

Green tick button is not getting activated through keyboard

User Impact:  When any UI component are not accessible by using keyboard then user who rely on keyboard are facing difficulty and not able to perform any action and are completely block to perform any functionality on that control element.

A11y Conformance Level: A

WCAG Success Criteria: 2.1.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4

**Comment by claudiahub** [comment]
@freben Hi Fredrick, 
this can be fixed by simply adding property `tabIndex=0` on the material UI icon as below code 
`<CheckCircle aria-label='status-ok' tabIndex=0 />` where CheckCircle is imported on backstage from `@material-ui/icons/CheckCircle` package. I guess this shouldn't be addressed on Backstage repo, thanks.
